service/azureconfig/v1/cloudconfig/render.go

```diff
@@ -59,9 +59,27 @@ func renderCertificatesFiles(certFiles certs.Files) ([]k8scloudconfig.FileAsset,
 }
 
 func renderCloudProviderConfFile(params cloudProviderConfFileParams) (k8scloudconfig.FileAsset, error) {
+	asset, err := renderCloudProviderConfFilePath(cloudProviderConfFileName, params)
+	if err != nil {
+		return k8scloudconfig.FileAsset{}, microerror.Mask(err)
+	}
+
+	return asset, nil
+}
+
+func renderCloudProviderConfFileMasterKubelet(params cloudProviderConfFileParams) (k8scloudconfig.FileAsset, error) {
+	asset, err := renderCloudProviderConfFilePath(cloudProviderConfFileNameMasterKubelet, params)
+	if err != nil {
+		return k8scloudconfig.FileAsset{}, microerror.Mask(err)
+	}
+
+	return asset, nil
+}
```

Comment:
This is BS but we have to provide separate cloud provider for master as long as it doesn't run in scale set.